frontend/src/components/DashboardDescriptionListGroup.tsx

```diff
@@ -99,7 +100,13 @@ const DashboardDescriptionListGroup: React.FC<DashboardDescriptionListGroupProps
         <DescriptionListTerm>{title}</DescriptionListTerm>
       )}
       <DescriptionListDescription className={isEmpty && !isEditing ? text.disabledColor_100 : ''}>
-        {isEditing ? contentWhenEditing : isEmpty ? contentWhenEmpty : children}
+        {isEditing ? (
+          contentWhenEditing
+        ) : isEmpty ? (
+          <Text style={{ color: '--pf-v5-global--Color--200' }}>{contentWhenEmpty}</Text>
```

Comment:
(Let's not block the PR, I can address this in my rebase of https://github.com/opendatahub-io/odh-dashboard/pull/2770)

This inline style shouldn't be necessary, that should be covered by the `className={isEmpty && !isEditing ? text.disabledColor_100 : ''}` above